MODULE.bazel

```diff
@@ -0,0 +1,18 @@
+module(
+    name = "rules_rust",
+    version = "0.9.0",
+)
+
+print("WARNING: The rules_rust Bazel module is still highly experimental and subject to change at any time. Only use it to try out bzlmod for now.")  # buildifier: disable=print
+
+bazel_dep(name = "platforms", version = "0.0.5")
+bazel_dep(name = "rules_cc", version = "0.0.1")
+bazel_dep(name = "bazel_skylib", version = "1.2.0")
+bazel_dep(name = "apple_support", version = "1.3.1")
+
+non_bzlmod_deps = use_extension("//rust/private:extensions.bzl", "non_bzlmod_deps")
+non_bzlmod_deps.non_bzlmod_deps()
```

Comment:
this line is unnecessary. the `use_extension` clause is enough to triger the extension

Reply:
Fixed